contracts/upgradeability/AdminUpgradeabilityProxy.sol

```diff
@@ -0,0 +1,175 @@
+/**
+ * SPDX-License-Identifier: MIT
+ *
+ * Copyright (c) 2018 zOS Global Limited.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+pragma solidity 0.6.8;
+
+import { UpgradeabilityProxy } from "./UpgradeabilityProxy.sol";
```

Comment:
Why did you copy these 3 proxy related contracts into the source tree instead of using the ones in `zos-lib`? At a quick glance they look very similar - were there changes you had to make?

Reply:
I just had to update the syntax to Solidity 0.6

Reply:
https://github.com/zeppelinos/zos-lib/blob/v1.3.0/contracts/upgradeability/AdminUpgradeabilityProxy.sol

Reply:
Addressed in https://github.com/centrehq/centre-tokens/pull/296/commits/bab078a50ad3083f5ab7d7b1c790a0bedf11b17e